redfish-core/lib/environment_metrics.hpp

```diff
@@ -243,11 +243,7 @@ inline void
 
                 asyncResp->res.jsonValue["PowerLimitWatts"]["SetPoint"] = 0;
                 asyncResp->res.jsonValue["PowerLimitWatts"]["ControlMode"] =
-                    "automatic";
-                asyncResp->res
-                    .jsonValue["PowerLimitWatts"]
-                              ["ControlMode@Redfish.AllowableValues"] = {
```

Comment:
Don't we still want AllowableValues? 
How does the client know what enum values we accept?

Reply:
I saw this too, but I thought AllowableValues might not be a valid property in the EnvironmentMetrics schema PowerWatts. https://redfish.dmtf.org/schemas/v1/EnvironmentMetrics.v1_1_0.json

Reply:
> I saw this too, but I thought AllowableValues might not be a valid property in the EnvironmentMetrics schema PowerWatts. https://redfish.dmtf.org/schemas/v1/EnvironmentMetrics.v1_1_0.json

Correct, so I removed this property.

Reply:
![screenshot](https://user-images.githubusercontent.com/5455127/148559779-9465f28e-c767-4118-9d22-b022a42727a9.JPG)
From the Redfish spec. It should be allowed on any property

Reply:
> ![screenshot](https://user-images.githubusercontent.com/5455127/148559779-9465f28e-c767-4118-9d22-b022a42727a9.JPG) From the Redfish spec. It should be allowed on any property

I checked the validator code:
https://github.com/DMTF/Redfish-Service-Validator/blob/master/common/catalog.py#L382-L391
Since the excerpt of the ControlMode attribute of Control_v1.xml does not contain `String`, the obtained excerptType = NEUTRAL

http://redfish.dmtf.org/schemas/v1/Control_v1.xml

```
<Property Name="ControlMode" Type="Control.v1_0_0.ControlMode">
<Annotation Term="OData.Permissions" EnumMember="OData.Permission/ReadWrite"/>
<Annotation Term="OData.Description" String="The current operating mode of the control."/>
<Annotation Term="OData.LongDescription" String="This property shall contain the operating mode of the control."/>
<Annotation Term="Redfish.Excerpt"/>
</Property>
```

So the validator will execute here:
https://github.com/DMTF/Redfish-Service-Validator/blob/master/validateRedfish.py#L34

@gtmills Please correct me if I am wrong, thanks!